packages/react-components/review/stable/react-components.api.md

```diff
@@ -97,6 +97,7 @@ export interface CameraButtonProps extends ControlBarButtonProps {
     enableDeviceSelectionMenu?: boolean;
     localVideoViewOptions?: VideoStreamOptions;
     onSelectCamera?: (device: OptionsDevice) => Promise<void>;
+    onShowVideoEffectsPicker?: (showVideoEffectsOptions: boolean) => void;
```

Comment:
nit: this should probably be onVideoEffectsClick and the implementation to show a picker onclick is not tied to the button but is composite behavior

Reply:
What does the argument `showVideoEffectsOptions` do again? (why is Options a boolean?)

Reply:
Changed the name to `onClickVideoEffects` and this boolean is used to show/hide the picker changed the name to `showVideoEffects`